src/main/java/vazkii/botania/common/core/handler/ConfigHandler.java

```diff
@@ -351,7 +351,10 @@ public static int loadPropPotionId(String propName, int default_) {
 	private static void verifyPotionArray() {
 		if(Loader.isModLoaded("DragonAPI"))
 			potionArrayLimit = Potion.potionTypes.length;
-		else potionArrayLimit = 127;
+		else if(Loader.isModLoaded("hodgepodge"))
```

Comment:
Is there a more detailed way to do this check, perhaps by actually reading some code to see if the mixin applied?

Especially since it's configurable in Hodge it would be nice to warn or error if it wasn't applied.